src/DIRAC/Core/Base/SQLAlchemyDB.py

```diff
@@ -9,7 +9,6 @@
 
 __RCSID__ = "$Id$"
```

Comment:
I guess all the occurrences of __RCSID__ can be deleted now as they are no filled in by the release procedure